src/styles/_nav.scss

```
@@ -0,0 +1,51 @@
+.nav {
+  text-transform: uppercase;
+  &__list {
```

Роби відступи між селекторами, так простіше читати
```suggestion
.nav {
  text-transform: uppercase;
  
  &__list {
```